### ISSUE TITLE
Minimal api scaffolding template - Commented code using wrong results type

### ISSUE THREAD

**Issue by mrward** [dotnet/Scaffolding#2074]
## File a bug

If you selected to 'Use TypedResults' the non-commented code correctly uses TypedResults, but the commented out code uses Results instead.

```
public static class MovieEndpoints
{
    public static void MapMovieEndpoints (this IEndpointRouteBuilder routes)
    {
        var group = routes.MapGroup("/api/Movie");

        group.MapGet("/", () =>
        {
            return new [] { new Movie() };
        })
        .WithName("GetAllMovies");
        
        group.MapGet("/{id}", (int id) =>
        {
            //return new Movie { ID = id };
        })
        .WithName("GetMovieById");

        group.MapPut("/{id}", (int id, Movie input) =>
        {
            return TypedResults.NoContent();
        })
        .WithName("UpdateMovie");

        group.MapPost("/", (Movie model) =>
        {
            //return Results.Created($"/Movies/{model.ID}", model);
        })
        .WithName("CreateMovie");

        group.MapDelete("/{id}", (int id) =>
        {
            //return Results.Ok(new Movie { ID = id });
        })
        .WithName("DeleteMovie");  
    }
}
```

These two sections seems to be the wrong way around.

https://github.com/dotnet/Scaffolding/blob/c5cbf3aa5086fbc22fbc5c820b1ae84c9459c66a/src/Scaffolding/VS.Web.CG.Mvc/Templates/MinimalApi/MinimalApi.cshtml#L95-L103

https://github.com/dotnet/Scaffolding/blob/c5cbf3aa5086fbc22fbc5c820b1ae84c9459c66a/src/Scaffolding/VS.Web.CG.Mvc/Templates/MinimalApi/MinimalApi.cshtml#L121-L128

### Include provider and version information

Microsoft.VisualStudio.Web.CodeGeneration.Design/Microsoft.DotNet.MSIdentity version:
Target framework: .NET 7.0
Operating system: macOS
IDE: Visual Studio for Mac 17.5

**Comment by deepchoudhery** [comment]
Thanks for catching that, fixed [here](https://github.com/dotnet/Scaffolding/pull/2153), will be serviced this week.

**Comment by deepchoudhery** [comment]
Fixed (7.0.1 and 6.0.11 package out today)
